requirements/edx/base.txt

```diff
@@ -129,7 +129,6 @@ help-tokens==1.1.2        # via -r requirements/edx/base.in
 html5lib==1.1             # via -r requirements/edx/base.in, ora2
 icalendar==4.0.7          # via -r requirements/edx/base.in
 idna==2.10                # via -r requirements/edx/paver.txt, requests
-importlib-metadata==1.7.0  # via -c requirements/edx/../constraints.txt, -r requirements/edx/paver.txt, kombu, path
```

Comment:
**Reviewer Notes:**
This is related to my interest in documentation for python environments. Why does this conflict with the output of `make upgrade` on Jenkins? Is it simple to have them match? Do we need improved documentation? Maybe it is that my devstack lms container is a little out of date, but I don't think so.

Can we approve this even though it conflicts with the Jenkins job? If not, my alternative is to just switch the constraint and let the job do the rest.  See https://github.com/edx/edx-platform/pull/25663/files.  However, this option is definitely not as nice and isn't always viable.

Reply:
See conversation from alternate ticket trying to determine the right way to update dependencies: https://github.com/edx/edx-platform/pull/25530#discussion_r528229113